extensions/ql-vscode/src/common/vscode/telemetry.ts

```diff
@@ -297,3 +297,21 @@ export async function initializeTelemetry(
   ctx.subscriptions.push(telemetryListener);
   return telemetryListener;
 }
+
+function isGlobalTelemetryEnabled(): boolean {
+  // If "enableTelemetry" is set to false, no telemetry will be sent regardless of the value of "telemetryLevel"
```

Comment:
When I have only `telemetry.enableTelemetry` set I get this popup:

![Screenshot 2023-09-15 at 16 53 16](https://github.com/github/vscode-codeql/assets/3749000/92418d19-5a64-4034-bebf-044e10de6f58)

That's where I'm getting the logic that if `telemetry.enableTelemetry` is set to `false` then it'll ignore the value of `telemetry.telemetryLevel`.